core/templates/dev/head/components/skills-mastery-list/skills-mastery-list.directive.ts

```diff
@@ -48,19 +45,29 @@ oppia.directive('skillsList', [
             ctrl.canCreateCollections = userInfo.canCreateCollections();
             ctrl.userIsLoggedIn = userInfo.isLoggedIn();
           });
+          ctrl.sortedSkillIds = [];
+
+          var sortSkillIdsByMastery = function() {
```

Comment:
Any reason why we don't want to do this in the html template itself using angular directives (i.e. orderBy)? This may cause more overhead compared to just using the built in directive to sort data right?

Reply:
Done.